module/VuFind/tests/unit-tests/src/VuFindTest/RecordDriver/EDSTest.php

```diff
@@ -99,7 +157,8 @@ public function testGetItemsAbstract(): void
      */
     public function testGetAccessLevel(): void
     {
-        $this->assertEquals('', $this->getDriver()->getAccessLevel());
+        $driver = $this->getDriver('valid-eds-record');
+        $this->assertFalse($driver->getAccessLevel());
```

Comment:
I notice that the `@return` annotation on getAccessLevel says it returns a string, but you're testing for a boolean here. Does the annotation need to be adjusted, or should the test be changed?

Reply:
Good catch, the annotation is correct so I went and fixed the test. I just double checked the EBSCO API documentation to be sure. In fact, I added the specific values that it could return in the function comment since it could be helpful to others looking at that function (I know I've had to go look back at the EBSCO documentation just to go find those values more than once).